go-fuzz-build/main.go

```diff
@@ -325,6 +325,9 @@ func (c *Context) buildInstrumentedBinary(blocks *[]CoverBlock, sonar *[]CoverBl
 	c.instrumentPackages(blocks, sonar)
 	mainPkg := c.createFuzzMain()
 	outf := c.tempFile()
+	if runtime.GOOS == "windows" {
+		os.Remove(outf)
```

Comment:
Do we rely on `c.tempFile()` result being an existent file anywhere else? If not we could remove it right in `tempFile`, which would be cleaner then polluting higher-level functions with OS-dependent quirks.